jac-uk/admin
# update jac Kit for Qt DL button

## What's included?
JAC-kit update amends the styling which was causing an issue which made the buttons un-clickable. 
This issue was raised specifically around QT response downloads.

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
- Navigate to the responses view of a QT.
- Check download button is clickable.

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
